web/src/pages/_app.tsx

```diff
@@ -24,6 +25,23 @@ function Account({ Component, pageProps: { ...pageProps } }: AppProps) {
         <TopNav />
         <Component {...pageProps} />
         <BottomNav />
+        <Toaster
+          position={"bottom-center"}
+          containerClassName={"!bottom-16 !select-none"}
+          gutter={16}
+          toastOptions={{
+            duration: 5000,
+            className: "!bg-slate-900/80 !text-slate-50 !backdrop-blur-md",
```

Comment:
What does `!` do in the tailwind class?

Reply:
!important in css

Reply:
This is needed to override some internal styling. I could use the headless version but it comes with no style and it would have taken me longer to customize it